services/minio/docker-compose.yaml.j2

```diff
@@ -39,12 +40,16 @@ services:
         - traefik.http.routers.minio.rule=PathPrefix(`/`)
         - traefik.http.routers.minio.entrypoints=storage
         - traefik.http.routers.minio.tls=true
-        - traefik.http.routers.minio.middlewares=gzip_compress@file
+        # Let's encrypt use-case
+        # - traefik.http.routers.minio.tls.certresolver=lehttpchallenge
+        - traefik.http.routers.minio.middlewares=gzip@docker
```

Comment:
Did you ever test performance impact?

Reply:
no. but it was also partly enabled in nginx. so I guess not much difference.